peps/pep-0730.rst

```diff
@@ -241,6 +239,44 @@ unique Framework name by using the full module name as the framework name (i.e.,
 ``foo.bar._whiz.framework``). A framework is a directory; the finder will look
 for ``_whiz.dylib`` in that directory.
 
+Compilation
+-----------
+
+The only binary format that will be supported is a dynamically-linkable
+``libpython3.x.dylib``, packaged in an iOS-compatible framework format. While
+the ``--undefined dynamic_lookup`` compiler option currently works, the
+long-term viability of the option cannot be guaranteed. Rather than rely on a
+compiler flag with an uncertain future, binary modules on iOS will be linked
+with ``libpython3.x``. This, in turn, makes statically-linkable builds of
+``libpython3.x.a`` impractical.
```

Comment:
```suggestion
with ``libpython3.x.dylib``. This, in turn, makes statically-linkable builds 
of ``libpython3.x.a`` impractical.
```

Reply:
I don't believe this suggestion is correct - or, at least, it doesn't capture what I was trying to communicate. The intention here is to say that binary modules will be linked with whatever library has been built. If you've built dynamic, then that's fine; but if you've built static, it *could* be linked into binary modules, but the resulting modules would be impractically large.

Reply:
I don't think an extension module could ever be linked against a static libpython, because that would cause duplication of globals. So the edit is technically unnecessary, but I thought it was better to be explicit.

Maybe something like this?
```
binary modules on iOS will be linked
with ``libpython3.x.dylib``. This, in turn, means they will not be loadable by an 
executable statically linked against ``libpython3.x.a``, so no such static library 
will be supported. This is the same pattern currently used by CPython on Windows.

For more details, see "Supporting static builds" below.
```

Reply:
Updated to reflect something close to this wording.